drivers/test.go

```diff
@@ -274,32 +275,44 @@ func (td *TestDriver) Complete() {
 	td.GasMeter.Record()
 }
 
-// TODO for failure cases we should consider catching panics here else they appear in the test output and obfuscate successful tests.
-func (td *TestDriver) ApplyMessageExpectReceipt(msg *types.Message, expected types.MessageReceipt) int64 {
+func (td *TestDriver) ApplyMessageExpectSuccess(msg *types.Message) int64 {
+	return td.ApplyMessageExpectSuccessAndReturn(msg, nil)
+}
+
+func (td *TestDriver) ApplyMessageExpectSuccessAndReturn(msg *types.Message, retval []byte) int64 {
+	return td.ApplyMessageExpectCodeAndReturn(msg, exitcode.Ok, retval)
+}
+
+func (td *TestDriver) ApplyMessageExpectCode(msg *types.Message, code exitcode.ExitCode) int64 {
+	return td.ApplyMessageExpectCodeAndReturn(msg, code, nil)
+}
+
+func (td *TestDriver) ApplyMessageExpectCodeAndReturn(msg *types.Message, code exitcode.ExitCode, retval []byte) int64 {
```

Comment:
Good stuff. Can we shorten the names even more? Suggest:
- ApplyOk
- ApplyExpect
- ApplyFailure
- (un-export, never useful to check both retval and exit code)